imperial.test.js

```diff
@@ -0,0 +1,376 @@
+import Imperial from "./imperial";
+import { Nation } from "./constants.js";
+import Action from "./action";
+
+describe("Imperial", () => {
+  describe("constructor", () => {
+    describe("six players", () => {
+      const state = {
```

Comment:
How do you feel about naming this variable `payload` instead? I think a name like that would help me keep in mind what type of thing this is. `state` doesn't sit right with me because it's really just one small chunk of what may be considered the game's state.

Alternatively, I guess we could name it `players` and make it refer to the array (not the object) and then we'd call it on line 18 below thusly:
```javascript
const game = Imperial.fromLog([Action.init({ players } )]);
```

The more I think about it, the more I like my second suggestion above. What do you think?